nio.env

```diff
@@ -11,22 +11,22 @@ INSTANCE_NAME: local
 INSTANCE_ID: local
 
 # Pubkeeper Client
-PK_HOST: 127.0.0.1
-PK_PORT: 9898
-PK_TOKEN: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwZXJtIjpbImFsbCJdLCJpc3MiOiJwdWJrZWVwZXIiLCJhdWQiOiJwdWJrZWVwZXIiLCJzdWIiOiJndWVzdCJ9.ODsw4iGEm4OeW10mqHEuMxqVOOZtITnETopee5GKb7M
-PK_SECURE: False
+PK_HOST: 
```

Comment:
can we make the default `your-system-id.pubkeeper.nio.works`, same for WS_HOST

Reply:
Should it be your-system-id.pubkeeper.nio.works or your-system-id.pubkeeper.nio.solutions?